frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -19,7 +19,11 @@ export const projectDropdown = {
       .contains(projectName)
       .click();
   },
-  shouldContain: (name: string) => cy.byLegacyTestID('namespace-bar-dropdown').contains(name),
+  shouldContain: (name: string) =>
+    cy
+      .byLegacyTestID('namespace-bar-dropdown')
+      .contains(name)
+      .should('exist'),
```

Comment:
nit, this should probably be the inverse of `shouldNotContain`:

```suggestion
      .should('contain', name),
```